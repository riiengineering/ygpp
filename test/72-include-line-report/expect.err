test/72-include-line-report/input:2: warning: root
test/72-include-line-report/inc/include1.in:1: warning: include 1
test/72-include-line-report/inc/include2.in:1: warning: include 2
test/72-include-line-report/inc/include2_1.in:1: warning: include 2.1
test/72-include-line-report/inc/include3.in:1: warning: include 3
test/72-include-line-report/inc/include3_1.in:1: warning: include 3.1
test/72-include-line-report/inc/include3_1_1.in:1: warning: include 3.1.1
test/72-include-line-report/inc/include3_1_2.in:1: warning: include 3.1.2
test/72-include-line-report/input:9: warning: end
