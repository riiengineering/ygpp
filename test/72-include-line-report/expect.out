This test tests nested includes including line reporting for warnings/errors.

This is included file 1.

This is included file 2.
This is included file 2.1.

This is included file 3.
This is included file 3.1.
This is included file 3.1.1.
This is included file 3.1.2.
